### PR TITLE
can-1277

### PR DESCRIPTION
**Pull request by shahab-ramzan**
first letter capital in activity logs

### CODE REVIEW

File: app/Helpers/TopicSupport.php [hunk 1]
```diff
@@ -1127,7 +1127,7 @@ public static function logActivityForRemoveCamps($removeCamps, $topicNum, $nickN
             $user = Nickname::getUserByNickName($nickNameId);
 
             $logType = "support";
-            $activity = "support removed";
+            $activity = "Support removed";
```
Comment (akuks) [inline app/Helpers/TopicSupport.php]: Please move it to common message file

File: app/Helpers/TopicSupport.php [hunk 2]
```diff
@@ -1127,7 +1127,7 @@ public static function logActivityForRemoveCamps($removeCamps, $topicNum, $nickN
...
             $model = new Support();
             $description = "supoort removed";
```
Comment (akuks) [inline app/Helpers/TopicSupport.php]: Please move it to common message file

File: app/Helpers/TopicSupport.php [hunk 3]
```diff
@@ -1168,7 +1168,7 @@ public static function logActivityForAddSupport($topicNum, $campNum, $nickNameId
             $campModel  = self::getLiveCamp($campFilter);
 
             $logType = "support";
-            $activity = "support added";
+            $activity = "Support added";
```
Comment (akuks) [inline app/Helpers/TopicSupport.php]: Please move it to common message file

File: app/Helpers/TopicSupport.php [hunk 4]
```diff
@@ -1168,7 +1168,7 @@ public static function logActivityForAddSupport($topicNum, $campNum, $nickNameId
...
             $link = Util::getTopicCampUrl($topicNum, $campNum, $topicModel, $campModel);
             $model = new Support();
             $description = "supoort added";
```
Comment (akuks) [inline app/Helpers/TopicSupport.php]: Please move it to common message file

File: app/Helpers/TopicSupport.php (hunk 2 above)
Comment (akuks) [reply app/Helpers/TopicSupport.php]: There is also a typo

**Review by akuks — CHANGES_REQUESTED** [review]

File: resources/lang/en/message.php [hunk 5]
```diff
@@ -53,7 +53,9 @@
     ],
     'general' => [
         'nickname_association_absence' => "Nickname not associated.",
-        'permission_denied' => "You don't have permission to access this resource."
+        'permission_denied' => "You don't have permission to access this resource.",
+        'support_added' => "Support added.",
```
Comment (akuks) [inline resources/lang/en/message.php:57]: Same for this

**Review by akuks — CHANGES_REQUESTED** [review]

File: resources/lang/en/message.php (hunk 5 above)
Comment (sajid315) [reply resources/lang/en/message.php:57]: Same for this means ? Messages are moved to language file where all other exists.

**Review by akuks — APPROVED** [review]

**Pull request merged**